pkgs/development/tools/nwjs/default.nix

```diff
@@ -32,11 +33,16 @@ in stdenv.mkDerivation rec {
   name = "nwjs-${version}";
   version = "0.32.1";
 
-  src = fetchurl {
+  src = if sdk then fetchurl {
     url = "https://dl.nwjs.io/v${version}/nwjs-v${version}-linux-${bits}.tar.gz";
```

Comment:
Now `sdk=true` downloads the normal version and `sdk=false` the sdk version. You sure? :smile: